soap/src/java/com/zimbra/soap/admin/message/CreateS3BucketConfigRequest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2010, 2011, 2012, 2013, 2014, 2016 Synacor, Inc.
```

Comment:
for new files/classes added keep only the current  year for copyright

Reply:
done!